packages/lesswrong/components/tagging/SubforumMembersDialog.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import { useMulti } from '../../lib/crud/withMulti';
+import DialogContent from '@material-ui/core/DialogContent';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  titleRow: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    columnGap: 14,
+    padding: '0 24px',
+  },
+  title: {
+    fontFamily: theme.typography.postStyle.fontFamily,
+    fontSize: 20,
+    lineHeight: '26px',
+    fontWeight: 400,
+    textTransform: 'capitalize'
+  },
+  joinBtn: {
+    '& button': {
+      minHeight: 0,
+      fontSize: 12,
+      padding: 6
+    }
+  },
+  user: {
+    marginBottom: 20
+  }
+})
+
+const SubforumMembersDialog = ({classes, onClose, tag}: {
```

Comment:
It's a little narrow on mobile, although I can see how making it wider would interfere with the clickaway to close behaviour

<img width="380" alt="Screenshot 2022-10-26 at 10 36 20" src="https://user-images.githubusercontent.com/77623106/197992546-1c32ad17-d354-4e81-9574-37886cbd8c88.png">

Reply:
Yeah I considered moving the images to the line above the username on mobile, but basically just wanted to wrap this up for now. I'm sure we'll tweak this more - I could imagine us removing all the bio and/or how I can help others text for example.